src/librustc_mir/Cargo.toml

```diff
@@ -9,13 +9,14 @@ path = "lib.rs"
 crate-type = ["dylib"]
 
 [dependencies]
+bitflags = "1.0"
 graphviz = { path = "../libgraphviz" }
 log = "0.3"
 rustc = { path = "../librustc" }
 rustc_const_eval = { path = "../librustc_const_eval" }
 rustc_const_math = { path = "../librustc_const_math" }
+rustc_cratesio_shim = { path = "../librustc_cratesio_shim" }
```

Comment:
This depends on libsyntax, so this isn't needed

Reply:
(copied from the other comment) True, but that feels fragile to me. What's the downside to always including this?